lib/data/landing_page_content_items/landing_page.yaml

```diff
@@ -0,0 +1,22 @@
+blocks:
+- type: paragraph
+  content: Here's some content!
```

Comment:
I was thinking we could use the [govspeak component](https://components.publishing.service.gov.uk/component-guide/govspeak) for things that are a bit long-form-text-like. That way we can keep our blocks to high level constructs, and avoid reproducing paragraphs / lists / tables etc.

Reply:
Gives the user a lot of flexibility (possibly too much, but I think it's better to err on that side of things for this project)

Reply:
Yeah, that makes sense. The paragraph was just for the proof of concept, I'll replace it with a govspeak block.

Reply:
Done

Reply:
Only remaining question here is for this version, do we allow govspeak in the content for that item or do we stick closer to what we'll get in the end from content store (HTML fragments converted from govspeak, which is what the component takes). I've chosen the latter path for now, I assume we generally do not want to be converting govspeak to HTML in the frontend.

Reply:
Hmm, that's a good question. It's a bit annoying to have to hardcode it as HTML, but I think that'll do for now. Options for later:

- Hardcode as HTML (current version)
- Hardcode as govspeak, have a build step to swap it out for HTML
- Use a real content item (publishing-api will render the govspeak)